plugins/azure/resoto_plugin_azure/resource/base.py

```diff
@@ -28,10 +36,14 @@ class AzureResource(BaseResource):
     # Which API to call and what to expect in the result.
     api_spec: ClassVar[Optional[AzureApiSpec]] = None
 
-    def delete(self, graph: Any) -> bool:
-        # TODO: implement me.
-        # get_client().delete(self.id)
-        return False
+    def delete(self, graph: GraphBuilder) -> bool:
```

Comment:
This method is called with `Graph` not `GraphBuilder`.
See AwsResource to get an idea of how to obtain the client.